test/built-ins/Temporal/Calendar/fields.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
```

Comment:
This should go in `test/built-ins/Temporal/Calendar/prototype/fields/`, but compare with the existing test https://github.com/tc39/proposal-temporal/blob/e057057bf18f81b226cd0d6b51faff3ac7da6ce7/polyfill/test/Calendar/prototype/fields/argument-iterable-not-array.js

Reply:
DONE